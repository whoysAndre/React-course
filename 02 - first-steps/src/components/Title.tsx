

interface IProps{
  title: string;
}
export default function Title({title}:IProps) {
  return (
    <>
      <h1>{title}</h1>
    </>
  )
}
