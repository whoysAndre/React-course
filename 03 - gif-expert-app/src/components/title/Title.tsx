
interface IProps{
  title: string;
}

export const Title = ({title}:IProps) => {
  return (
    <h1 className="text-3xl font-bold">
      {title}
    </h1>
  )
};
