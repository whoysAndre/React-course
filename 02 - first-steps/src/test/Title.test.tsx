import {  render } from "@testing-library/react"
import Title from "../components/Title";

describe('Title', () => { 
  
  test('Debemos hacer un snapshop', () => {
    
    render(<Title title="Hello world"/>)
    
    //Primero desestrucutaremos el render 
    const { container } = render(<Title title="Hello world"/>);

    expect(container).toMatchSnapshot();


  });
  
})
