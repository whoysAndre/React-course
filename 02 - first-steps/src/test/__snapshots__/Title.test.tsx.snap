// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Debemos hacer un snapshop 1`] = `
<div>
  <h1>
    Hello world
  </h1>
</div>
`;
